shuup/gdpr/utils.py

```diff
@@ -96,32 +99,39 @@ def ensure_gdpr_privacy_policy(shop, force_update=False):
             with create_revision():
                 gdpr_document = Page.objects.create(
                     shop=shop,
-                    page_type=PageType.REVISIONED,
                     content=content,
                     available_from=now_date,
                     title=force_text(_("Privacy Policy")),
-                    url=force_text(_("privacy-policy"))
+                    url=settings.GDPR_PRIVACY_POLICY_PAGE_URLS.get(current_language, "privacy-policy")
```

Comment:
I guess this would become unmaintainable in the long-run. Adding a new entry every time a new language set would be bad. Maybe check if the translation for the current language exists, if not, append the language code to the fallback language, something like:

```py
url = slugify(force_text(_("Privacy Policy")))
# pseudo-code
if not has_translations(current_language, url):
  url = "{}-{}".format(url, current_language)  # if there is no such translation, then the url will be `privacy-policy-it`, for Italian, for example

```

Reply:
Thing is, we don't want URL's to be translatable